GTG/gtk/plugins.py

```diff
@@ -128,7 +128,7 @@ def __init__(self, requester):
         builder.add_from_file(ViewConfig.PLUGINS_UI_FILE)
 
         self.dialog = builder.get_object("PluginsDialog")
-        self.dialog.set_title(_(f"Plugins - {info.NAME}"))
+        self.dialog.set_title(_("Plugins"))
```

Comment:
You can keep the details if you want with something like
`self.dialog.set_title(_("Plugins - %s") % info.NAME)`, preferably with a translator comment above it, stating that the variable is the plugin name.

Reply:
Yup, in the case at hand I did this change simply because I didn't see the point of showing the version number (or app name) in the window title nowadays, and this lets me reuse the same string that is used in the menubutton so one less translation to deal with.